### ISSUE TITLE
Windows sound effect when the wiimote is connected

### ISSUE THREAD

**Issue by Naxo175** [simphax/Touchmote#104]
Hello! When my wiimote is connected, there is this windows sound effect (3m11s) https://youtu.be/sRTvrtuuGJQ?t=191
It is repeated roughly all the 10 seconds. What can I do ?

**Comment by Hansel-G** [comment]
Yes this is an issue I just recently had, Wiimote keeps disconnect and re-connect every 10 seconds by itself, gets worse with a nunchuk on it'll repeat every 5 seconds. Works before it suddenly has issues
